### ISSUE TITLE
Multiple delegations does not update total delegation

### ISSUE THREAD

**Issue by MessiasMorten** [oasisprotocol/oasis-wallet-ext#167]
Hi!

I have had a stake of 23000 Rose tokens delegated to a validator for a while. I recently bought more, and decided to stake it towards the same validator. However, the UI does not update my delegation, and it views as if I never staked my second round at all.

![Total delegation on the main tab](https://i.imgur.com/qY84l3W.png)

![Total delegation in the staking tab](https://i.imgur.com/Snxc2X2.png)

**Comment by pro-wh** [comment]
Thanks for the report. I see that we have some code to update the UI after a transfer. It would definitely make sense to update it after escrow related operations too.

**Comment by MessiasMorten** [comment]
Sounds good. Furthermore, I would like to add that this bug also extends partially to the web wallet, and in fact to Oasisscan as well. Thanks for looking into it :)

**Comment by pro-wh** [comment]
I got some more info from another user, and they said it took some time to update. The wallet should reload enough, but now we think there was some delay in the block explorer API in this case.

**Comment by MessiasMorten** [comment]
Yeah, I noticed that too. It took somewhere between 8 hours and 2 days for it to update. I didn't really pay attention to it past those 8 hours, so it could be anywhere within that span. I still find that timeline excessive.
